content/fr/docs/reference/kubectl/kubectl.md

```diff
@@ -11,7 +11,7 @@ kubectl contrôle le manager d'un cluster Kubernetes
 
 kubectl contrôle le manager d'un cluster Kubernetes. 
 
-Vous trouverez plus d'informations ici : https://kubernetes.io/docs/reference/kubectl/overview/
+Vous trouverez plus d'informations ici : /docs/reference/kubectl/overview/
```

Comment:
ça ne va pas rendre comme un lien. Je pense qu'un lien markdown est souhaitable ici

Reply:
yeah, you are right, I am going to fix it.
![image](https://user-images.githubusercontent.com/1580956/54828147-eeac2380-4c89-11e9-8adb-71e5c0b2dca9.png)

Reply:
but my concern is if we put back the full link, it might still look weird as well.

The original doc in here might need some fix.

sorry, I am not french speaker.

Reply:
same thing for this, `https://kubernetes.io/fr/`
https://kubernetes.io/fr/docs/reference/kubectl/overview/

Reply:
Yes that's right :)